### PR TITLE
Creates Homepage

### PR DESCRIPTION
**Pull request by seemavora**
image
image
image

### CODE REVIEW

File: src/pages/homepage/home.css [hunk 1]
```diff
@@ -1,38 +1,109 @@
...
+.homepage-img{
+  height: 500px;
+  width: 85%;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.banner-text{
+  /* background-color:rgb(250, 248, 148); */
```
Comment (ThaiQ) [inline src/pages/homepage/home.css]: Remove

File: src/pages/homepage/home.css [hunk 2]
```diff
@@ -1,38 +1,109 @@
...
+  background-color: rgb(6, 0, 59);
+  height: auto;
+  opacity:0.8;
+  width: 100%;
+  margin:auto;
+  /* border-radius:30px; */
```
Comment (ThaiQ) [inline src/pages/homepage/home.css]: Remove

File: src/pages/homepage/home.css [hunk 3]
```diff
@@ -1,38 +1,109 @@
...
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.banner-text h1 {
+  font-size: 66px;
+  font-weight: bold;
+  /* color: rgb(4, 35, 134); */
```
Comment (ThaiQ) [inline src/pages/homepage/home.css:26]: Remove

File: src/pages/homepage/home.css [hunk 4]
```diff
@@ -1,38 +1,109 @@
...
+  color: rgb(255, 255, 255);
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.banner-text hr {
+  /* border-top: 5px dotted  rgb(4, 35, 134); */
```
Comment (ThaiQ) [inline src/pages/homepage/home.css]: Remove

File: src/pages/homepage/home.css [hunk 5]
```diff
@@ -1,38 +1,109 @@
...
+  border-top: 5px dotted  rgb(255, 255, 255);
+  color: rgb(255, 255, 255);
+  widows: 100%;
+  margin:auto;
+}
+
+.banner-text p{
+  /* color: rgb(4, 35, 134); */
```
Comment (ThaiQ) [inline src/pages/homepage/home.css]: Remove

File: src/pages/homepage/home.css [hunk 6]
```diff
@@ -1,38 +1,109 @@
...
+  color: rgb(255, 255, 255);
+  font-size: 35px;
+  font: Comic Sans;
+  padding:25px 50px 75px;
+  line-height: 100%;
+  
+}
+
+.meet-team {
+  /* display: flex;
```
Comment (ThaiQ) [inline src/pages/homepage/home.css]: Remove

File: src/pages/homepage/home.css [hunk 7]
```diff
@@ -1,38 +1,109 @@
...
+}
+
+.meet-header {
+  color: rgb(4, 35, 134);
+  font-size: 50px;
+  text-decoration-line: underline overline;
+  font-weight: bold;
+  margin: 0 auto;
+  text-align: center;
+  /* flex-direction: column;
```
Comment (ThaiQ) [inline src/pages/homepage/home.css]: Remove

File: src/pages/homepage/home.css [hunk 8]
```diff
@@ -1,38 +1,109 @@
...
+  display: -moz-flex;
+  display: -webkit-flex;
+  display: flex;
+}
+
+.wrap-reverse         { 
+  -webkit-flex-wrap: wrap-reverse;
+  flex-wrap: wrap;
+  justify-content: center;
+}
```
Comment (ThaiQ) [inline src/pages/homepage/home.css]: empty lines

File: src/pages/homepage/home.js [hunk 9]
```diff
@@ -1,27 +1,99 @@
...
+                for these students.
+            </p>
+            </div>
+            <div className='meet-header'>
+              Meet the Team
+            </div>
+            <p className='info'>Our team includes 5 young developers who hope to make a positive difference
+              using their technical skills. This platform runs entirely on the IBM cloud.
+            </p>
+            {/* <div className='meet-team'> */}
```
Comment (ThaiQ) [inline src/pages/homepage/home.js]: Remove

File: src/pages/homepage/home.js [hunk 10]
```diff
@@ -1,27 +1,99 @@
...
+              <ul className="flex-container wrap-reverse">
+                {creatorCards}
+              </ul>
+
+            {/* </div> */}
```
Comment (ThaiQ) [inline src/pages/homepage/home.js]: Remove

File: src/pages/homepage/home.js [hunk 11]
```diff
@@ -1,27 +1,99 @@
...
+import TeamCard from './teamCard'
 import UserNavbar from '../../components/navbar/navbar';
+import nanar from '../homepage/pictures/nanar.jpeg';
+import annabel from '../homepage/pictures/annabel.jpeg';
+import seema from '../homepage/pictures/seema.jpeg'
+import surabhi from '../homepage/pictures/surabhi.png'
+import thai from '../homepage/pictures/thai.png'
+
+
+class home extends Component {
```
Comment (ThaiQ) [inline src/pages/homepage/home.js]: Capital "Home"

also do this:
```js
export default class Home extends Component{
```

File: src/pages/homepage/home.js [hunk 12]
```diff
@@ -1,27 +1,99 @@
...
+
+
+          </div>
+        </div>
+      </div>
+    )
+  }
 }
+
+export default home
```
Comment (ThaiQ) [inline src/pages/homepage/home.js]: You don't need this if you fixed the "class home" comment above

File: src/pages/homepage/teamCard.css [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+  font-size: 18px;
+}
+
+.card-wrapper {
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+  flex-wrap: wrap;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  text-align: center;
+  min-width: 400px;
+  /* width: 400px;
```
Comment (ThaiQ) [inline src/pages/homepage/teamCard.css]: remove

File: src/pages/homepage/teamCard.js [hunk 14]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {CardImg, Card, CardBody, CardTitle, CardText} from 'reactstrap';
+
+import './teamCard.css';
+
+
+class TeamCard extends React.Component {
```
Comment (ThaiQ) [inline src/pages/homepage/teamCard.js]: same

File: src/pages/homepage/teamCard.js [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+      </Card>
+    </div>
+
+
+     
+    );
+  }
+}
+
+export default TeamCard
```
Comment (ThaiQ) [inline src/pages/homepage/teamCard.js]: same

**Review by ThaiQ — CHANGES_REQUESTED** [review]

**Review by nanarboursal — APPROVED** [review]
love it!!

**Review by Annabel1054 — APPROVED** [review]
YAY GOOD JOB!
WAIT! In the 'About Us' section, you spelled deserves wrong. Make sure to fix that!

**Review by surabhigupta2029 — APPROVED** [review]
Looks great Seema!

**Review by ThaiQ — APPROVED** [review]

**Pull request merged**